### ISSUE TITLE
Accessibility Research: How should the app behave when a new form is added

### ISSUE THREAD

**Issue by jeromeleecms** [CMSgov/eAPD#3985]
In many parts of our app, there is the option to add new form elements. 

Example:

![image.png](https://images.zenhubusercontent.com/5c547884d11206601037f0c7/2b527a70-bf32-45c6-9a91-a0f0161ae2d0)

When clicking on add outcome, a new text area appears:

![image.png](https://images.zenhubusercontent.com/5c547884d11206601037f0c7/b4f278ba-4377-4c67-9ec9-3a501746ad3c)

What should the app do from a:
- Screen reader perspective (should the app announce the new form?)
- Focus perspective (should the focus on the app move to the new text field (1) or stay on the outcomes section (2)?)

### This task is done when...

- [ ] We have an answer on the screen reader question
- [ ] We have an answer on the form focus question

**Comment by itsonlydio** [comment]
1. From a screen reader perspective the app should announce when a new form is generated
2. The focus on SHOULD move to the newly generated field as it was created by the user input.
